### PR TITLE
changing targets number from 5 to 7

### PR DESCRIPTION
**Pull request by tongju12**
## Scientists have 7 targets in PA1K4:PF instead of 5 on hands. 

Change target number from 5 to 7
## Motivation and Context

Scientists request to have 7 targets instead of 5 in PA1K4:PF motor

## How Has This Been Tested?

Not yet because hardware are not ready.

## Where Has This Been Documented?

(https://jira.slac.stanford.edu/browse/ECS-4824)

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive docstrings, including context and API
- [ ] New/changed functions and methods are covered in the test suite where possible
- [ ] Test suite passes locally
- [ ] Test suite passes on GitHub Actions
- [ ] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [ ] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: docs/source/upcoming_release_notes/1286-adding_more_targets_into_pa1k4.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+----------
+- N/A
+
+Library Features
+----------------
+- N/A
+
+Device Features
+---------------
+- Adding two more targets in device
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1286-adding_more_targets_into_pa1k4.rst:14]:
```suggestion
- Changes `CalibrationAxis` from 5 -> 7 states
```

File: docs/source/upcoming_release_notes/1286-adding_more_targets_into_pa1k4.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- @tongju12
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1286-adding_more_targets_into_pa1k4.rst:30]:
```suggestion
- tongju12
```

**Review by tangkong — APPROVED** [review]
This seems ok, as long as you've verified this is correct for your configuration.  `CalibrationAxis` is only used in your `SCaFoil` class so far.

**Review by tangkong — CHANGES_REQUESTED** [review]
actually a few changes, not an approval just yet

File: docs/source/upcoming_release_notes/1286-adding_more_targets_into_pa1k4.rst (hunk 1 above)
Comment (tongju12) [reply docs/source/upcoming_release_notes/1286-adding_more_targets_into_pa1k4.rst]: I just changed and push new doc. Thanks!

**Review by tangkong — APPROVED** [review]

**Pull request merged**